openbook_notifications/views.py

```diff
@@ -52,9 +56,13 @@ def post(self, request):
         data = serializer.validated_data
 
         max_id = data.get('max_id')
+        types = data.get('types')
+
+        if types is not None:
```

Comment:
After the previous comments are solved, the valid types should be already on a list in the data.get('types')